### ISSUE TITLE
Unable to find node on an unmounted component for React 17

### ISSUE THREAD

**Issue by praWINK** [weknowinc/react-bubble-chart-d3#37]
When I tried to implement with react 17 it throws Unable to find a node on an unmounted component at the line ReactDom.findDOMNode(this) in react-bubble-chart-d3.js file

**Comment by reginz** [comment]
yes, I am getting the same error right now! have you be able to solve this?

**Comment by charmjeh** [comment]
+1

**Comment by shankar4569** [comment]
Yep majority got this error

**Comment by Shweta-themesbrand** [comment]
I am also facing this issue in React 17. Are there any updates on this? @praWINK

**Comment by jakemwood** [comment]
Just so folks know who find this in the future, this project has been forked a few times to make this React 17 compatible, our fork included: https://github.com/TestBoxLab/react-bubble-chart-d3. It's also on npm under `@testboxlab/react-bubble-chart-d3`

**Comment by shoaib171** [comment]
I face Error when install this package and use code to set bubble charts then that  page crashed. How i can use and alternative please support
